_data/master/navbars/security.yml

```diff
@@ -44,3 +44,5 @@ toc:
     path: /security/comms/reduce-nodes
   - title: Securing Calico's Prometheus endpoints
     path: /security/comms/secure-metrics
+- title: Admin policy using domain names
```

Comment:
I suggest "Use domain names in policy rules" as the title.  It's a little more "How too..." if you see what I mean.

Reply:
How to is good, but I think the title also needs to make clear that this is admin-only.  Would "Use domain names in admin policy rules" work for you?

Reply:
"admin" means different things to different personas. e.g. if you were to ask someone what the difference between an "admin policy rule" and a "policy rule" is, I think you would get different answers from different personas.  One of the aims with the how-to titles is to be persona neutral.